test/releasers/yoshi-go.ts

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {describe, it, before} from 'mocha';
+import * as nock from 'nock';
+import {GoYoshi} from '../../src/releasers/go-yoshi';
+import {readFileSync} from 'fs';
+import {resolve} from 'path';
+import * as snapshot from 'snap-shot-it';
+
+const fixturesPath = './test/releasers/fixtures/yoshi-go';
+
+interface MochaThis {
+  [skip: string]: Function;
+}
+function requireNode10(this: MochaThis) {
```

Comment:
What is happening here?  Why are we specifically trying to add logic to avoid running tests on node 8 and below?

Reply:
copy pasta, we can get rid of this.